### DISCUSSION TITLE
Regionlar orqali districtlarni qanday chaqirish mumkin

### DISCUSSION THREAD

**Discussion by qodirbozorov** [MIMAXUZ/uzbekistan-regions-data#10] [General]
Bizga biror viloyatni tanlaganda uning ichidagi tumanlarni chiqaradigan object kerak edi, bunda taaluqlilik ketma ketligi yo'q ekna

**Comment by MIMAXUZ** [comment]
Ha albatta ola olasiz. Har bir viloyatga uning tumanlari bog'langan. Kichik shaharchalar va qishloqlar esa tumanlarga bog'langan. Biz umumiy sxemani shakllantirganmiz. Agar ma'lumotlar bazasi bilan ishlay olsangiz so'rov yozib chiqaza olasiz...
